usr/sbin/p11sak/p11sak.c

```diff
@@ -912,17 +913,17 @@ static CK_ULONG char2attrtype(char c)
     }
 }
 
-static void set_bool_attr_from_string(CK_ATTRIBUTE *attr, char *attr_string)
+static void set_bool_attr_from_string(CK_ATTRIBUTE *attr, char attr_string)
 {
```

Comment:
Please check: is attr_string only a char instead of a ptr?

Reply:
It is.  See changes in this commit below.